helm/app/templates/dashboard-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Instead of creating new template for secrets use existing one and configure it from values file

https://github.com/vardius/go-api-boilerplate/blob/05d918d6c6df9fc969f7b4df6d597e5313a126ff/helm/app/values.yaml#L251